tools/api-markdown-documenter/src/test/snapshots/markdown/simple-suite-test/flat-config/test-suite-a.md

```diff
@@ -1017,6 +1018,51 @@ testConstWithEmptyDeprecatedBlock: string
 
 # Namespace Details
 
+## TestBetaNamespace {#testbetanamespace-namespace}
+
+A namespace tagged as `@beta`.
+
+**WARNING: This API is provided as a beta preview and may change without notice. Use at your own risk.**
+
+### Signature {#testbetanamespace-signature}
+
+```typescript
+export declare namespace TestBetaNamespace
+```
+
+### Remarks {#testbetanamespace-remarks}
+
+Tests release level inheritance.
+
+### Variables
+
+| Variable | Alerts | Modifiers | Type | Description |
+| --- | --- | --- | --- | --- |
+| [betaMember](docs/test-suite-a#testbetanamespace-betamember-variable) | `Beta` | `readonly` |  |  |
+| [publicMember](docs/test-suite-a#testbetanamespace-publicmember-variable) | `Beta` | `readonly` |  |  |
```

Comment:
`alphaMember` isn't supposed to show up here? I see it in the doc above but don't quite get why not here.

Reply:
This document is part of a different suite from the file above. This config only yields documentation for items that are `@beta`+.